### ISSUE TITLE
Cant catch any pokemon

### ISSUE THREAD

**Issue by keith5995** [finndev/PokeBuddy#753]
does someone know how to fix it

**Comment by Skolzack** [comment]
If you can't catch any pokemon there's a 3 reasons:

1 - Your inventory is full.
2 - You don't have pokeballs.
3 - You're soft banned. (also you can't get EXP spinning pokestops too)

**Comment by keith5995** [comment]
No. I still can get exp spinning pokestops,  I can see pokestops. But,  it won't catch any pokemon .I also check my bag, it is not full and I hv  pokeballs
